README.md

```diff
@@ -1,9 +1,46 @@
-# Merch Embedded 
-
+# Merch Embedded
 
 [![Join the chat at https://gitter.im/acm-uiuc/merch-development](https://badges.gitter.im/acm-uiuc/merch-development.svg)](https://gitter.im/acm-uiuc/merch-development?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Possible information sources:
+
+This repository contains all of the code related to controlling the underlying merch hardware (the vending machine).
+Merch runs an embedded webserver that can be accessed at merch.acm.illinois.edu.
+
+## API
+
+Every request through the API has a token in the header that is unique to groot.
+This way, only groot can make requests to vend things.
+The token is stored in the request header.
+
```

Comment:
Should be something like 

"Requests to the device must contain a valid token in the Authorization Header for a request to be processed. 
As of now the only token will be given solely to the groot merch service, so if you wish to make merch requests go through groot. "

Also rename the header field from Token to Authorization, better match with web conventions (may require change by @sameetandpotatoes on the service side)